task/main.py

```diff
@@ -11,13 +11,17 @@
 import psycopg2
 from psycopg2 import sql
 
-
 def get_db_params():
     sm = boto3.client('secretsmanager')
     secrets_arn = os.getenv('CUMULUS_CREDENTIALS_ARN', None)
     secrets = json.loads(sm.get_secret_value(SecretId=secrets_arn).get('SecretString'))
 
-    db_params = {'sslmode': 'disable'} # Will revisit when/if SSL becomes required
+    query_timeout_offset = 1000
+    statement_timeout_ms = int(os.getenv("QUERY_TIMEOUT", 900)) * 1000 - query_timeout_offset
```

Comment:
`os.getenv("QUERY_TIMEOUT", 900)`
may be a little redundant since the environment variable should be populated with the terraform default variable (both are currently 900)
Not necessarily an issue but may create confusion for instance, if this default value is updated to something like `1000000` but is not being used (because the environment variable should always be auto-set by terraform)